docs/administration.rst

```diff
@@ -456,111 +397,11 @@ when you have lots of obsoleted packages, would be like so:
-  while true; do ps aux | grep runner; sleep 3; clear; done # wait until there are no runner processes
-
-First check that this specific version of openstack is available via:
-
-.. code-block:: bash
-
-  openstack --os-volume-api-version 3.42 volume list
-
-The command should not fail.
-
-To resize a volume:
-
-.. code-block:: bash
-
-  # get the 'openstack' volume id
-  juju storage --volume # the volume id is in the "Provider ID" column
-  # from the above command, get the id, and set it to a variable: VOLUME_ID
-  openstack --os-volume-api-version 3.42 volume set ${VOLUME_ID} --size ${NEW_SIZE}
-  # this will begin the process of resizing the volume
-  # whilst this is happening, consider running this:
-  while true; do openstack volume show ${VOLUME_ID}; sleep 5; clear; done
-  # If the volume in question has been retyped (__DEFAULT__ <-> Ceph_NVMe), run the following (not necessary for volumes that haven't been retyped):
-  nova reboot ${server_name}
-  # where $server_name is the name of the server associated with the volume
-  # to check this:
-  juju storage # make note of the juju unit name associated with the storage you've resized
-  # then
-  openstack server list
-  # and get the server name of the server running the unit mentioned in juju storage
-  # after rebooting, run the following ON THE SERVER you've rebooted
-  lsblk # check that the disk size has increased
-  sudo growpart /dev/vdb 1
-  sudo resize2fs /dev/vdb1
-  lsblk # check that the disk size and partition sizes match
-
-There are no conclusions as to why the reboot is required if the volume has already
-been retyped. None of the typical methods for rescanning disks work, in this case.
-
-When the volume hasn't been retyped prior, it is immediately acknowledged by the
-openstack server. Keep this in mind if you're using the __DEFAULT__ volume type
-(see `openstack volume show ${VOLUME_ID}` to check).
-
 
 Killing running tests
 ---------------------
 
-In order to kill a currently running test, grab the test uuid. This can be seen in
-`running.json` or on the `/running` page.
-
-`ssh` to a worker unit, and run:
-
-.. code-block:: bash
-
-  ps aux | grep runner | grep $uuid
-  # grab the PID from the process - this approach will also remove the test request from the queue
-  kill -9 $pid
-  # if you want to stop or restart the test but preserve the test request, run the following to get the service name:
-  service_name=$(ps aux | grep runner | grep $uuid | grep -oP '(?<=--security-groups).*?(?=--name)' | cut -d'@' -f2 | sed -e "s/.secgroup//g")
-  # and then simply stop or restart as you please:
-  sudo systemctl restart/stop autopkgtest@"${service_name}".service
-
-This will kill the autopkgtest process, and then the worker will `ack` the test request
-message, causing the test to not be re-queued, and then the worker will also ensure
-that the openstack server used for the test is deleted.
+TODO: replicate this functionality
```

Comment:
I'll take the occasion to mention that we should also implement a "graceful shutdown" functionality for the workers, i.e. finish the current test and terminate.